### ISSUE TITLE
The competitor analyses spreadsheet mentioned on "Winning Clients - Discovery" is missing

### ISSUE THREAD

**Issue by marcusx** [wunderkraut/WunderWay#73]
On http://way.wunder.io/working-with-clients/winning-new-clients/winning-clients-discovery/ there is a link to  "TOOL: Competitor Analysis Worksheet" but it is missing and a search in drive didN#t bring up anything like that.

**Comment by marcusx** [comment]
I change the title. It seem on all places where a "TOOL" in drive is linked I get an "Sorry, the file you have requested does not exist."

**Comment by joesb** [comment]
I'll take this one, see if I can work out where these links should go

**Comment by joesb** [comment]
Hmm, actually this might need @steveparks or @huntons to look at these broken links - I'm not sure what I'm trying to find in Google Drive. :(

**Comment by huntons** [comment]
I've already asked Heikki and SteveP for their input on this one

On Tue, Jul 14, 2015 at 4:24 PM, Joe Baker <notifications@github.com> wrote:

> Hmm, actually this might need @steveparks <https://github.com/steveparks>
> or @huntons <https://github.com/huntons> to look at these broken links -
> I'm not sure what I'm trying to find in Google Drive. :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wunderkraut/WunderWay/issues/73#issuecomment-121281289>
> .
>



-- 
Steve Hunton
UK Projects & Operations Manager
Wunder UK

Check out our Healthy Web Projects <http://wunder.uk/healthy-web-projects>

email: steve.hunton@wunder.uk <steve.hunton@wunderroot.com>
phone: 00 44 20 3588 0181

http://www.wunder.uk
Nurturing Growth and Success in Digital Business.

**Comment by badrange** [comment]
Links are still missing. @heikkiyrjola @steveparks help?

**Comment by steveparks** [comment]
Aha - these were documents created by @WunderLemon and there was a point where some folders in her google drive were deleted rather than moved, by a quirk of google drive UI. So maybe Alex has copies, or she and I can work to reconstruct them?

**Comment by WunderLemon** [comment]
Sorry for slowness in replying to this. It slipped my notice.
I remember through the mists of time what these docs were about in general, but I'm afraid I don't have anything for them anymore.
When I stopped Wunderworking for a while in June 2014, everything attached to my Wunder Google account was erased. I never had these docs on my hard drive either.
I'd recognise them if I saw them again, but I think they might be lost to the ether unless anyone else can dig them out…

**Comment by steveparks** [comment]
Yes unfortunately there was a problem with the way google docs handles transferring ownership of a whole folder of documents.
So - unfortunately we need to re-create the documents. @WunderLemon can you ping me in hipchat to schedule a time next week to work on this together?

**Comment by WunderLemon** [comment]
@steveparks I'm tracking the docs back to source – i.e. I've emailed Amy Nicholls to see if they originated with her. She's going to dig around her hard drive tonight. I'll report back…
